.github/workflows/release.yml

```
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "*"
```

```suggestion
on:
  push:
    branches:
      - "main"
    tags:
      - "v*"
```

We don't need CD for other branches
Let's also run CD when there is a new version tag